src/main/com/intellij/lang/jsgraphql/ide/introspection/GraphQLIntrospectionService.java

```diff
@@ -146,9 +146,17 @@ public void actionPerformed(@NotNull AnActionEvent e, @NotNull Notification noti
         try {
             final GraphQLSettings graphQLSettings = GraphQLSettings.getSettings(myProject);
             String query = buildIntrospectionQuery(graphQLSettings);
-
-            final String requestJson = "{\"operationName\": \"IntrospectionQuery\", \"query\":\"" + StringEscapeUtils.escapeJavaScript(query) + "\"}";
-            HttpPost request = createRequest(endpoint, url, requestJson);
+            HttpPost request;
+            if (graphQLSettings.isOperationNameEnabled()) {
```

Comment:
if setting is enabled, we check the keyname user wants in graphql settings or default to "operationName" then use default operation name value "IntrospectionQuery" since that would be appropriate value in most cases
`{"operationName": "IntrospectionQuery", "query": {}}`

Reply:
Can also make this dynamic like the normal request if there is a use case, can't think of one unless the graphql server is configured to only accept Introspection Query with a specific operation name